content/en/docs/tasks/tools/install-kubectl-macos.md

```diff
@@ -123,10 +123,10 @@ The following methods exist for installing kubectl on macOS:
    kubectl version --client --output=yaml
    ```
 
-1. After installing the plugin, clean up the installation files:
+1. After installling and validating kubectl, delete the checksum file:
```

Comment:
The suggested changes contains typos. Please review and correct them.

```suggestion
1. After installing and validating kubectl, delete the checksum file:
```

Reply:
Now  i have updated based on the feedback. Thanks for review.